ios/Integreat.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
+   <FileRef
+      location = "group:../android/app/src/main/assets/fonts/Lateef.ttf">
+   </FileRef>
+   <FileRef
+      location = "group:../android/app/src/main/assets/fonts/OpenSans-Bold.ttf">
+   </FileRef>
+   <FileRef
+      location = "group:../android/app/src/main/assets/fonts/OpenSans-Regular.ttf">
+   </FileRef>
+   <FileRef
+      location = "group:../android/app/src/main/assets/fonts/Raleway-Bold.ttf">
+   </FileRef>
+   <FileRef
+      location = "group:../android/app/src/main/assets/fonts/Raleway-Regular.ttf">
+   </FileRef>
```

Comment:
Here's something wrong I guess. Seems like I've imported these files not the right way. Can you check this on the mac @maxammann?

Reply:
Not right now, but I think you can revert all the changes in the workspacedata